docs/plugins/discovery-ec2.asciidoc

```diff
@@ -182,60 +230,139 @@ Management Console. It should look similar to this.
+|`_ec2:publicDns_`   |The public host of the machine.
+|`_ec2:privateIp_`   |Equivalent to `_ec2:privateIpv4_`.
+|`_ec2:publicIp_`    |Equivalent to `_ec2:publicIpv4_`.
+|`_ec2_`             |Equivalent to `_ec2:privateIpv4_`.
+|==================================================================
+
+These values are acceptable when using any discovery type. They do not require
+you to set `discovery.seed_providers: ec2`. However they do require that the
+`discovery-ec2` plugin is installed.
+
 [[cloud-aws-best-practices]]
 ==== Best Practices in AWS
 
-Collection of best practices and other information around running Elasticsearch on AWS.
+This section contains some other information about designing and managing an
+{es} cluster on your own AWS infrastructure. If you would prefer to avoid these
+operational details then you may be interested in a hosted {es} installation
+available on AWS-based infrastructure from http://www.elastic.co/cloud.
+
+===== Storage
+
+EC2 instances offer a number of different kinds of storage. Please be aware of
+the folowing when selecting the storage for your cluster:
+
+* http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/InstanceStorage.html[Instance
+Store] is recommended for {es} clusters as it offers excellent performance and
+is cheaper than EBS-based storage. {es} is designed to work well with this kind
+of ephemeral storage because it replicates each shard across multiple nodes. If
+a node fails and its Instance Store is lost then {es} will rebuild any lost
+shards from other copies.
 
-===== Instance/Disk
-When selecting disk please be aware of the following order of preference:
+* https://aws.amazon.com/ebs/[EBS-based storage] may be acceptable
+for smaller clusters (1-2 nodes). Be sure to use provisioned IOPS to ensure
+your cluster has satisfactory performance.
 
-* https://aws.amazon.com/efs/[EFS] - Avoid as the sacrifices made to offer durability, shared storage, and grow/shrink come at performance cost, such file systems have been known to cause corruption of indices, and due to Elasticsearch being distributed and having built-in replication, the benefits that EFS offers are not needed.
-* https://aws.amazon.com/ebs/[EBS] - Works well if running a small cluster (1-2 nodes) and cannot tolerate the loss all storage backing a node easily or if running indices with no replicas. If EBS is used, then leverage provisioned IOPS to ensure performance.
-* http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/InstanceStorage.html[Instance Store] - When running clusters of larger size and with replicas the ephemeral nature of Instance Store is ideal since Elasticsearch can tolerate the loss of shards. With Instance Store one gets the performance benefit of having disk physically attached to the host running the instance and also the cost benefit of avoiding paying extra for EBS.
+* https://aws.amazon.com/efs/[EFS-based storage] is not
+recomended or supported as it does not offer satisfactory performance.
```

Comment:
spelling

Reply:
Fixed in 1dda1297a34bd69ad5afe4f9507d7434149f6ec9.